include/deal.II/distributed/tria.h

```diff
@@ -286,6 +286,14 @@ namespace parallel
 
       typedef typename dealii::Triangulation<dim,spacedim>::CellStatus CellStatus;
 
+      /**
+       * This auxiliary data structure stores the relation between a p4est quadrant,
+       * a deal.II cell and the current CellStatus assigned across mesh refinement
+       * as a tuple.
+       */
+      typedef typename std::tuple< typename dealii::internal::p4est::types<dim>::quadrant *,
+              CellStatus, cell_iterator > quadrant_cell_relation_t;
+
```

Comment:
Keep this typedef in the public interface? Move it somewhere else? Or abandon it completely?

Reply:
I don't think this is used outside this class, so it should at best be part of the private interface.